### ISSUE TITLE
Warning Message for Line Endings

### ISSUE THREAD

**Issue by pplavetzki** [microsoft/docs-scaffolding#33]
## Description

When the scaffolding completes it adds incorrect line endings for Mac.

```
warning: CRLF will be replaced by LF in learn-pr/azure/aks-azure-monitor-prometheus/1-introduction.yml.
The file will have its original line endings in your working directory
warning: CRLF will be replaced by LF in learn-pr/azure/aks-azure-monitor-prometheus/2-learning-content.yml.
The file will have its original line endings in your working directory
warning: CRLF will be replaced by LF in learn-pr/azure/aks-azure-monitor-prometheus/3-exercise.yml.
The file will have its original line endings in your working directory
warning: CRLF will be replaced by LF in learn-pr/azure/aks-azure-monitor
```

## To Reproduce

1. On Mac/Linux create a new Learn module.
2. Commit the changes you will receive the above warnings.

**Comment by jamarw** [comment]
Hi @pplavetzki , thanks for reporting this. We'll take a look.

**Comment by jamarw** [comment]
Hi @pplavetzki , found the issue and I'm working on a fix now. Should be released in the next day or two. Thanks!

**Comment by jamarw** [comment]
https://github.com/microsoft/docs-scaffolding/pull/38

**Comment by jamarw** [comment]
Hi @pplavetzki, this issue is resolved in release 0.0.5. Thanks!
